README.md

```diff
@@ -1,5 +1,23 @@
-# **Plugin Name** Plugin
+# Discourse hCaptcha Plugin
 
-**Plugin Summary**
+## Overview
 
-For more information, please see: **url to meta topic**
+This plugin is designed to integrate HCaptcha into the sign-up form of Discourse forums. This plugin aims to enhance security and bot protection by leveraging the privacy-centric features of HCaptcha. The setup process is straightforward and consists of a few easy steps.
+
+## Installation
+
+1. **Create an HCaptcha Account**:
+   - Visit [HCaptcha](https://www.hcaptcha.com/) to create an account. After registering, you'll receive a site key and a secret key.
+
+2. **Setup Local Testing** (Optional):
+   - If you are testing locally, add a new virtual host entry to your hosts file. Include a line like `127.0.0.1 test.mydomain.com`. Make sure the domain is valid, even if you don't own it, to ensure the HCaptcha script loads properly.
+
+3. **Configure Plugin Settings**:
+   - Log into your Discourse admin panel.
+   - Navigate to `Admin` > `Settings` > `Plugins` > `hCaptcha (settings)`.
+   - In this section, add the site key and secret key you obtained from HCaptcha.
+
+4. **Update Content Security Policy**:
```

Comment:
There's an `extend_content_security_policy` API which you can use from `plugin.rb` to do this automatically

(see https://github.com/discourse/discourse/commit/488fba3c5f747db7783424b05e1f4190bb494a2b)